modules/kernel/src/main/java/org/opencastproject/kernel/security/SystemTokenRememberMeUtils.java

```diff
@@ -44,10 +44,19 @@ public final class SystemTokenRememberMeUtils {
 
   private static final Logger logger = LoggerFactory.getLogger(SystemTokenRememberMeUtils.class);
 
+  /** This is the default cookie key, that is configured in Opencast **/
+  private static final String defaultCookieKey = "opencast";
+
   private SystemTokenRememberMeUtils() {
   }
 
   public static String augmentKey(String key) {
+
+    if (!defaultCookieKey.equals(key)) {
+      logger.debug("The default cookie key 'opencast' is not in use. The given key won't be augmented.");
```

Comment:
Since `defaultCookieKey` exists, we shouldn't hard-code the value:

```suggestion
      logger.debug("The default cookie key '{}' is not in use. The given key won't be augmented.", defaultCookieKey);
```

Reply:
Done